packages/cmr-client/CMRSearchConceptQueue.js

```diff
@@ -24,22 +25,24 @@ class CMRSearchConceptQueue {
    * The constructor for the CMRSearchConceptQueue class
    *
    * @param {Object} params
-   * @param {string} params.provider - the CMR provider id
-   * @param {string} params.clientId - the CMR clientId
+   * @param {string} params.cmrSettings - the CMR settings for the requests - the provider,
+   * clientId, and either launchpad token or EDL username and password
    * @param {string} params.type - the type of search 'granule' or 'collection'
    * @param {string} params.cmrEnvironment - optional, CMR environment to
    *              use valid arguments are ['OPS', 'SIT', 'UAT']
    * @param {string} [params.searchParams={}] - the search parameters
    * @param {string} params.format - the result format
    */
   constructor(params = { searchParams: {} }) {
-    this.clientId = params.clientId;
-    this.provider = params.provider;
+    this.clientId = params.cmrSettings.clientId;
+    this.provider = params.cmrSettings.provider;
```

Comment:
It looks like `this.provider` was only used in the constructor and we're not even using it there on line 41 anymore, so do we even need it?

Reply:
Yes, it is used in the search params when the actual search is performed.

Reply:
Unless I'm missing something, it looks like the search only uses `this.params` and `this.params` is not set using `this.provider` but `params.cmrSettings.provider`. This isn't critical though

Reply:
Ah yeah those don't need to be set. I removed clientId too and switched over `searchConcept` to use the CMR function.